requirements-dev.txt

```diff
@@ -1,7 +1,7 @@
-pytest
+pytest>=2.8
```

Comment:
`pytest-flake8` has a minimum dependency and travis was failing because it had an older version.